pkg/scrape/navr.go

```diff
@@ -23,6 +27,7 @@ func NaughtyAmericaVR(knownScenes []string, out *[]ScrapedScene) error {
 		colly.AllowedDomains("www.naughtyamerica.com"),
 		colly.CacheDir(sceneCacheDir),
 		colly.UserAgent(userAgent),
+		colly.Async(true),
```

Comment:
Why this one is async?

Reply:
LOL.. cuz I had a ton of changes in flight and missed it ;)

I wondered why it completed so fast.